### PR TITLE
feat: cart added

### PR DESCRIPTION
**Pull request by som749898**
Please review my component
Screenshot (85)

### CODE REVIEW

**Review by rohanmathur91 — APPROVED** [review]
LGTM

File: Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 5px;
+}
+
```
Comment (TheAmlanSaikia) [inline Cart/cart.css:7]: You can add a utility class for flex.

File: Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+.cart-btn {
+  width: 100%;
+}
+
+.cart-text {
+  padding: 5px 0;
+}
+
```
Comment (TheAmlanSaikia) [inline Cart/cart.css:15]: Use of rem is preferred

**Review by ranitasahaa1230 — APPROVED** [review]
Good to merge

**Review by TheAmlanSaikia — APPROVED** [review]
LGTM. Left a few comments.

File: Cart/cart.css (hunk 1 above)
Comment (som749898) [reply Cart/cart.css:7]: I will add these in the component library after completing ecom screens

File: Cart/cart.css (hunk 2 above)
Comment (som749898) [reply Cart/cart.css:15]: Vikram mentioned below 10px we need not convert to rem

**Pull request merged**